### PR TITLE
Add SITL swarming guide

### PR DESCRIPTION
**Pull request by Ryanf55**
Coming out of https://github.com/ArduPilot/ardupilot/issues/20161#issuecomment-1842083930

The dev team put their heads together and we found the right commands to do it.

### CODE REVIEW

File: dev/source/docs/using-sitl-for-ardupilot-testing.rst [hunk 1]
```diff
@@ -214,6 +214,43 @@ is equivalent to COM16:
...
+
+The other way to spawn the vehicles is the swarm configuration file.
+An example configuration is found in ``Tools/autotest/swarminit.txt``, with the file format described in the header.
+This allows configurable ENU offsets as well as an initial absolute heading for each vehicle.
+
+::
+
+    sim_vehicle.py -v Copter --map --console --count 5 --auto-sysid --swarm Tools/autotest/swarminit.txt
+
+:ref:`MavProxy's Multiple Vehicle Guide <multi>` contains information to control the swarm.
```
Comment (Ryanf55) [inline dev/source/docs/using-sitl-for-ardupilot-testing.rst:252]: This doesn't link. Any syntax tips?

Comment (Hwurzburg) [reply dev/source/docs/using-sitl-for-ardupilot-testing.rst:253]: I would do the MAVProxy separate...merge it...to establish the links inventory, then the ref in this doc can be tested locally

Comment (Ryanf55) [reply dev/source/docs/using-sitl-for-ardupilot-testing.rst]: Done.

Comment (Hwurzburg) [reply dev/source/docs/using-sitl-for-ardupilot-testing.rst]: I've already done the change and merged it....I think the wiki on the server needs to rebuild for the inventory to update, but will trying building this now anyway

Comment (Hwurzburg) [reply dev/source/docs/using-sitl-for-ardupilot-testing.rst]: yep will have to wait for wiki to build to really test the link to the MAVProxy mult page....

Comment (Ryanf55) [reply dev/source/docs/using-sitl-for-ardupilot-testing.rst]: Ok. Want to merge this as-is, then add the cross-ref later?

Comment (Hwurzburg) [reply dev/source/docs/using-sitl-for-ardupilot-testing.rst]: it can wait a day

**Pull request merged**